src/App.js

```diff
@@ -1,11 +1,28 @@
 import React from "react";
+import { BrowserRouter, Routes, Route, Navigate } from "react-router-dom";
+
 import { AppBlock } from "./styled";
-import Notes from "pages/Notes";
+import MainMenu from "pages/MainMenu/MainMenu";
+import Notes from "pages/MyNotes/Notes";
+import SharedNotes from "pages/SharedNotes/SharedNotes";
+import About from "pages/About/About";
+import NotFound from "pages/NotFound/NotFound";
```

Comment:
Let's define index.js to avoid duplicating inside imports
index.js should import inside component/container and export it at the same time. Result will be:
```suggestion
import MainMenu from "pages/MainMenu";
import Notes from "pages/MyNotes";
import SharedNotes from "pages/SharedNotes";
import About from "pages/About";
import NotFound from "pages/NotFound";
```

Reply:
Also MainMenu is not a page!!!! It's a shared reusable component!